br/pkg/task/backup.go

```diff
@@ -137,7 +137,7 @@ func (cfg *BackupConfig) ParseFromFlags(flags *pflag.FlagSet) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	cfg.BackupTS, err = ParseTSString(backupTS)
+	cfg.BackupTS, err = ParseTSString(backupTS, false)
```

Comment:
Why does backup full-snapshot use false?

Reply:
Because We don't want to make breaking change here.